ossm/merge_upstream.sh

```diff
@@ -49,13 +52,22 @@ merge() {
   return $?
 }
 
-main () {  
+updateVersionsInOssmValuesYaml() {
+    latest_version=$(yq '.versions[0].version' < "${VERSIONS_YAML_FILE}")
+    minor_version=${latest_version%.*}
+    latest_version_underscore=${latest_version//./_}
+    minor_version_underscore=${minor_version//./_}
+    sed -i -e "s/${minor_version}\.[0-9]\+/${latest_version}/g" -e "s/${minor_version_underscore}_[0-9]\+/${latest_version_underscore}/g" "${HELM_VALUES_FILE}"
+}
+
+main () {
   if ! merge; then
     set -e
     echo "Conflicts detected, attempting to run 'make gen' to resolve."
     rm -rf bundle/**/*.yaml resources bundle.Dockerfile
+    updateVersionsInOssmValuesYaml
```

Comment:
Also we don't want to run this for new minor versions, only for new micro versions